ethdb/prune/storage_mode.go

```diff
@@ -137,6 +154,9 @@ type Mode struct {
 type Distance uint64
 
 func (p Distance) Enabled() bool { return p != math.MaxUint64 }
+func (p Distance) toValue() uint64 { return uint64(p) }
+func (p Distance) useDefaultValue() bool { return uint64(p) == params.FullImmutabilityThreshold }
+func (p Distance) dbType() []byte { return kv.PruneHistoryTypeOlder }
```

Comment:
probably need rename `kv.PruneHistoryTypeOlder` to `kv.PruneTypeOlder`, because "history" word reserved for "historical state access" meaning.

Reply:
Changes in https://github.com/ledgerwatch/erigon-lib/pull/76